scripts/nb-tester/qiskit_docs_notebook_tester/__init__.py

```diff
@@ -373,210 +65,13 @@ def cancel_trailing_jobs(start_time: datetime, config_path: str) -> bool:
-        action="store_true",
-        help=(
-            "Specify if the qiskit generic backend should enable control flow"
-            "directives on the target"
-        )
-    )
-
-    runtime_options_group = parser.add_argument_group(
-        "qiskit-ibm-runtime options",
-        description=(
-            "These options change the behavior when --provider=qiskit-ibm-runtime, "
-            "and are passed as parameters directly to QiskitRuntimeService. "
-            "See https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.QiskitRuntimeService "
-            "for more details."
-        )
-    )
-    runtime_options_group.add_argument(
-        "--channel",
-        action="store",
-        default="ibm_quantum",
-        choices=["ibm_cloud", "ibm_quantum", "local"],
-        help=(
-            "Specify a channel for running the notebook against."
-        )
-    )
-    runtime_options_group.add_argument(
-        "--token",
-        action="store",
-        default=None,
-        help=(
-            'IBM Cloud API key or IBM Quantum API token. Warning: for security,' 
-            'you should set this via an environment variable, e.g. '
-            '`--token="${IQP_API_TOKEN}".'
-        )
-    )
-    runtime_options_group.add_argument(
-        "--url",
-        action="store",
-        default=None,
-        help=(
-            "The API URL to submit the notebook against."
-        )
-    )
-    runtime_options_group.add_argument(
-        "--name",
-        action="store",
-        default=None,
-        help=(
-            "Name of the qiskit account to load."
-        )
-    )
-    runtime_options_group.add_argument(
-        "--instance",
-        action="store",
-        default=None,
-        help=(
-            "The service instance to use."
-        )
-    )
-    args = parser.parse_args()
-    if args.only_submit_jobs:
-        args.submit_jobs = True
-    return args
-
-
-async def _main() -> None:
-    args = get_args()
-    config = Config.from_args(args)
-    paths = list(config.notebooks_to_execute())
-
-    if not paths:
-        print("⚠️ No notebooks to execute - exiting")
-        return
-
-    if paths and args.is_fork:
-        print(
-            "⛔️ We can't run notebook tests on pull requests from forks becaus of how GitHub Secrets work."
-            "\n\n"
-            "If you have write access to Qiskit/documentation, push to a new "
-            "branch there and make your pull request from that branch instead."
-            "\n\n"
-            "If you don't have write access, you should locally test out the notebooks you're modifying "
-            "by using the instructions in "
-            "https://github.com/Qiskit/documentation#execute-notebooks. "
-            "When this PR is approved, a maintainer will merge it to a new "
-            "branch in Qiskit/documentation, then make a PR from that branch "
-            "into main so it can pass CI.\n",
-        )
```

Comment:
Good catch. It'd be helpful to keep this.

Reply:
Good spot! I plan to move that functionality to our GitHub action because it's very specific to our repo, but it shouldn't have been removed in this PR.

Reply:
It used to live there, but we decided to move it to the script in https://github.com/Qiskit/documentation/pull/2062. I'm fine with moving this back to the GH action, so long as it preserves the same prior functionality of not failing when the notebooks couldn't have been tested anyways.

Reply:
I remember that, here's what I had in mind. It is the kind of thing I'd forget so thanks for reminding me.

https://github.com/Qiskit/documentation/blob/9dbd215c8e31db1c69e7dcd1b069c6bf47fe2b04/.github/workflows/notebook-test.yml#L71-L88